.github/workflows/continuous_integration.yml

```
@@ -122,7 +122,7 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ secrets.FAIRY_TOKEN }}
           message: |
-            🎉 Ta-daaa, freshly created APKs are available:
+            🎉 Ta-daaa, freshly created APKs are available for https://github.com/${{ github.repository }}/pull/${{ github.event.pull_request.number }}/commits/${{ github.sha }}:
```

```suggestion
            🎉 Ta-daaa, freshly created APKs are available for ${{ github.sha }}:
```

I think that should do the same and be shorter to read (only showing the first 7 chars from the sha would be even better)

---

Let's try it.

---

btw, it is already showing the first 7 chracters in the comments?

---

@m-kuhn okay, works, ready to merge :)